data/resource.csv

```diff
@@ -0,0 +1,2 @@
+ID,NAME
+<RESOURCE_ID>,<RESOURCE_NAME>
```

Comment:
This `data/` folder is redundant you can remove it and all the files inside it. This was never cleaned up when designing the files view